sbc_translation/wizards/reply_to_comments.py

```diff
@@ -0,0 +1,14 @@
+from odoo import models, fields, api
+
+
+class ReplyToComments(models.TransientModel):
+    _name = "reply.to.comments"
+
+    paragraph_ids = fields.Many2many("correspondence.paragraph", string="Paragraphs", readonly=True)
+    answer = fields.Text(string="Answer")
```

Comment:
Html will enable better editing of the reply. The string attribute is not needed if it's the same name as the field.
```suggestion
    answer = fields.Html()
```